[2023-07-16T01:34:12.101+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main_dag.tsk1 manual__2023-07-16T01:34:10.680743+00:00 [queued]>
[2023-07-16T01:34:12.111+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main_dag.tsk1 manual__2023-07-16T01:34:10.680743+00:00 [queued]>
[2023-07-16T01:34:12.111+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-07-16T01:34:12.126+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): tsk1> on 2023-07-16 01:34:10.680743+00:00
[2023-07-16T01:34:12.130+0000] {standard_task_runner.py:57} INFO - Started process 2379 to run task
[2023-07-16T01:34:12.133+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'main_dag', 'tsk1', 'manual__2023-07-16T01:34:10.680743+00:00', '--job-id', '419', '--raw', '--subdir', 'DAGS_FOLDER/main_dag.py', '--cfg-path', '/tmp/tmpr8l0ufm6']
[2023-07-16T01:34:12.134+0000] {standard_task_runner.py:85} INFO - Job 419: Subtask tsk1
[2023-07-16T01:34:12.178+0000] {task_command.py:410} INFO - Running <TaskInstance: main_dag.tsk1 manual__2023-07-16T01:34:10.680743+00:00 [running]> on host 3be1cdcf9db5
[2023-07-16T01:34:12.270+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='main_dag' AIRFLOW_CTX_TASK_ID='tsk1' AIRFLOW_CTX_EXECUTION_DATE='2023-07-16T01:34:10.680743+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-16T01:34:10.680743+00:00'
[2023-07-16T01:34:12.272+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-07-16T01:34:12.272+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install networkx']
[2023-07-16T01:34:12.282+0000] {subprocess.py:86} INFO - Output:
[2023-07-16T01:34:13.003+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-07-16T01:34:13.169+0000] {subprocess.py:93} INFO - Requirement already satisfied: networkx in /home/***/.local/lib/python3.7/site-packages (2.6.3)
[2023-07-16T01:34:18.025+0000] {subprocess.py:93} INFO - 
[2023-07-16T01:34:18.026+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.1.2 -> 23.2
[2023-07-16T01:34:18.026+0000] {subprocess.py:93} INFO - [notice] To update, run: python -m pip install --upgrade pip
[2023-07-16T01:34:18.181+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-07-16T01:34:18.212+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=main_dag, task_id=tsk1, execution_date=20230716T013410, start_date=20230716T013412, end_date=20230716T013418
[2023-07-16T01:34:18.245+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-07-16T01:34:18.265+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
